src/models/anchor.ts

```diff
@@ -1,9 +1,19 @@
-export class Anchor {
-  id!: number
-  requestId!: string
-  path!: string
-  cid!: string
-  proofCid!: string
-  createdAt!: Date
-  updatedAt!: Date
-}
+import { type, string, TypeOf } from 'codeco'
+import { cidAsString, date } from '../ancillary/codecs.js'
+
+export const FreshDatabaseAnchor = type({
```

Comment:
What do you think about calling this type `UnpersistedAnchor`?  I feel like it's a bit confusing calling it a `DatabaseAnchor` when its not yet in the database

Reply:
I changed the names similar to how they are for metadata. We have `StoredMetadata` and `FreshMetadata`. Similarly, now these two are called `StoredAnchor` and `FreshAnchor`. Is it all right for you?

Reply:
yeah that sounds good, thanks